src/argilla_sdk/records/_dataset_records.py

```diff
@@ -302,6 +306,38 @@ def to_list(self, flatten: bool = False) -> List[Dict[str, Any]]:
         """
         return self(with_suggestions=True, with_responses=True).to_list(flatten=flatten)
 
+    def to_disk(self, path: str) -> str:
+        """
+        Export the records to a file on disk. This is a convenient shortcut for dataset.records(...).to_disk().
+
+        Parameters:
+            path (str): The path to the file to save the records.
+            orient (str): The structure of the exported dictionary.
+
+        Returns:
+            The path to the file where the records were saved.
+
+        """
+        record_dicts = self(with_suggestions=True, with_responses=True).to_list()
+        with open(path, "w") as f:
+            json.dump(record_dicts, f)
+        return path
+
+    def from_disk(self, path: str) -> "DatasetRecords":
+        """Creates a DatasetRecords object from a disk path.
+
+        Args:
+            path (str): The path to the dataset records.
+
+        Returns:
+            DatasetRecords: The DatasetRecords object created from the disk path.
+
+        """
+        with open(path, "r") as f:
+            records = json.load(f)
+        self.update(records=[_dict_to_record(record) for record in records])
```

Comment:
I would expect to use this to add records. but this is what the `update` does if records do not exits.

Reply:
Yes, but `add` does not match upsert, so it can create confusing behavior if the method is run multiple times.

Reply:
Yes, maybe it makes sense to rename `update` to `upsert` in any case